content/glossary.md

```diff
@@ -3,19 +3,52 @@ title: Glossary
 kind: Documentation
 draft: true
 ---
+
 **Agent**
 
+**API**
+
+Datadog provides an HTTP API in order to interact programmatically with the platform. Every feature, resource, mechanism, and bit of functionality can be accessed via the API. Interacting with the API manually is a great way to learn how Datadog works under the hood! In production however, you will use a purpose-built tool, library, or interface — such as the Datadog product itself — as an abstraction layer between you and the API.
```

Comment:
and bit of functionality feels weird. can we leave it at '... , and mechanism can be accessed...'

Reply:
don't think manually is needed. 'interacting with' i think implies manually

Reply:
And maybe not ending with an !

Reply:
> and bit of functionality feels weird. can we leave it at '... , and mechanism can be accessed...'

Agree

> don't think manually is needed. 'interacting with' i think implies manually

Disagree. There are lots of ways to interact with the API that aren't cURL.

> And maybe not ending with an !

Party-pooper. :P

Reply:
then you are interacting with the framework/library that is interacting with the api.